File: java/0002_jewels-and-stones.java
class Solution {
    public int numJewelsInStones(String J, String S) {
        int len_S = S.length();
        
        int count = 0;
        
        String arr[] = S.split("");
        for(int i=0; i<len_S; i++){
            
            if(J.indexOf(arr[i])!=-1){
                count += 1;
            }
            
        }
        
        return count;
    }
}

File: python/0002_jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        
        S_unique = [x for x in S]
        S_unique = list(S_unique)
        
        
        count = 0
        
        for ele in S_unique:
            for temp in [y for y in J]:
                if ele == temp:
                    count += 1
                else:
                    continue
                    
        return count
